scripts/core/ServiceOsmApiDbHootApiDbConflate.sh

```diff
@@ -59,7 +59,7 @@ HOOT_DB_URL="hootapidb://$DB_USER:$DB_PASSWORD@$DB_HOST:$DB_PORT/$DB_NAME"
 # generic debugging options applicable to multiple commands
 HOOT_OPTS="--warn -D uuid.helper.repeatable=true -D writer.include.debug.tags=true"
 if [ "$CONFLATION_TYPE" == "network" ]; then
-  HOOT_OPTS=$HOOT_OPTS" -D match.creators=hoot::NetworkMatchCreator -D merger.creators=hoot::NetworkMergerCreator -D network.matcher=hoot::ConflictsNetworkMatcher -D conflate.match.highway.classifier=hoot::HighwayExpertClassifier -D way.subline.matcher=hoot::MaximalSublineMatcher -D rubber.sheet.minimum.ties=4 -D rubber.sheet.ref=true"
+  HOOT_OPTS=$HOOT_OPTS" -D match.creators=hoot::NetworkMatchCreator -D merger.creators=hoot::NetworkMergerCreator -D network.matcher=hoot::ConflictsNetworkMatcher -D conflate.match.highway.classifier=hoot::HighwayExpertClassifier -D way.subline.matcher=hoot::MaximalSublineMatcher -D rubber.sheet.minimum.ties=4 -D rubber.sheet.ref=true -D poi.polygon.address.match.enabled=false"
 fi
```

Comment:
As noted above, disabled poi/poly address matching in a number of tests.  Just left it enabled in the following:

PoiPolyStandalone
PoiPolygConflate
ConflateExact
PoiPolyAddressScoreExtractor
a couple of the poi-polygon cases tests

If I forgot to remove it anywhere, then we can fix later.